src/screens/Dashboard/CompactTickets.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {useState} from 'react';
+import {StyleSheet} from '@atb/theme';
+import * as Sections from '@atb/components/sections';
+import {
+  ShortTicketInfoView,
+  mapToUserProfilesWithCount,
+} from '../Ticketing/Ticket/TicketInfo';
+import {isPreactivatedTicket, isInspectableTicket} from '@atb/tickets';
+import {getValidityStatus} from '@atb/screens/Ticketing/Ticket/utils';
+import useInterval from '@atb/utils/use-interval';
+import {
+  useHasEnabledMobileToken,
+  useMobileTokenContextState,
+} from '@atb/mobile-token/MobileTokenContext';
+import {findReferenceDataById} from '@atb/reference-data/utils';
+import {useFirestoreConfiguration} from '@atb/configuration/FirestoreConfigurationContext';
+import {
+  filterActiveOrCanBeUsedFareContracts,
+  isValidRightNowFareContract,
+  useTicketState,
+} from '@atb/tickets';
+import ThemeText from '@atb/components/text';
+import {TicketsTexts, useTranslation} from '@atb/translations';
+
+type Props = {
+  onPressDetails?: (orderId: string) => void;
+};
+
+const CompactTickets: React.FC<Props> = ({onPressDetails}) => {
+  const itemStyle = useStyles();
+
+  // TODO: Verify if this is the correct way to update time-to-time the expiration time of the tickets.
+  const [now, setNow] = useState<number>(Date.now());
+  useInterval(() => setNow(Date.now()), 1000);
+
+  const {fareContracts, customerProfile} = useTicketState();
+  const activeFareContracts = filterActiveOrCanBeUsedFareContracts(
+    fareContracts,
+  ).sort(function (a, b): number {
+    const isA = isValidRightNowFareContract(a);
+    const isB = isValidRightNowFareContract(b);
+
+    if (isA === isB) return 0;
+    if (isA) return -1;
+    return 1;
+  });
+
+  // TODO: Double check if this is how it should be implemented, using the customer profile!
+  const hasActiveTravelCard = !!customerProfile?.travelcard;
```

Comment:
This is a required parameter in order to reuse the main component here, however, I saw this in other places, but I want to make sure it is correct here in this context.

Reply:
Looking at the logic inside `isInspectableTicket()`, this is only used when mobile token is not enabled, so this looks correct.

Reply:
We talked about moving all interactions on travelcard over to checking tokens instead of customer profiles. However, the webshop at least hasn't done that transition. But that is a improvement over time